sfxclient/gometrics.go

```diff
@@ -59,3 +65,47 @@ func (g *goMetrics) Datapoints() []*datapoint.Datapoint {
 		Gauge("num_goroutine", dims, int64(runtime.NumGoroutine())),
 	}
 }
+
+func (g *goEvents) Events() []*event.Event {
```

Comment:
Looked at this with fresh eyes.  Gometrics is used by our tests for sample datapoints, but it does serve as a mechanism for collecting system metrics on its own.  That is why it was broken out separately.  Since the events I created were purely as test data, I have moved the events to "httpsink_test.go".